### PR TITLE
jj4x4: README update

### PR DESCRIPTION
**Pull request by mariusrugan**
## Description
README clarification for JJ4X4 Reset key location. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jj4x4/README.md [hunk 1]
```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
 
     make jj4x4:default:flash
 
-**Reset Key**: Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard.
+**Reset Key**:
 
-See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard usb cable.
```
Comment (Rionlion100) [inline keyboards/jj4x4/README.md]:
```suggestion
Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard USB cable.
```

File: keyboards/jj4x4/README.md [hunk 2]
```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
...
+
+Key *K00* is in the above picture:
+
+Row 3 from bottom (or 1 from top), Column 2 from left (or 3 from right), just like the schematic below:
```
Comment (Rionlion100) [inline keyboards/jj4x4/README.md]:
```suggestion
Row 2 from the top, Column 2 from the left, see the schematic below:
```
This is optional but makes more sense in my opinion

**Comment by fauxpark** [comment]
Per the layout macro define in jj4x4.h, K00 (actually K01 since the columns are 1-indexed in this particular macro) is the top left key, not `8`. Which is correct? Is this a Bootmagic thing, or part of bootloadHID?

File: keyboards/jj4x4/README.md (hunk 2 above)
Comment (mariusrugan) [reply keyboards/jj4x4/README.md]: sure, it's more simple.

**Comment by mariusrugan** [comment]
> Per the layout macro define in jj4x4.h, K00 (actually K01 since the columns are 1-indexed in this particular macro) is the top left key, not `8`. Which is correct? Is this a Bootmagic thing, or part of bootloadHID?

Hi thanks for looking into this, and bare with me please as i am a QMK noob.

Correct is my description, 
(that's why the pull request, due to the fact that i forgot the magic key position and i had to re-dig the internet around to find it) 

This key is "builtin" by the vendor, KRepublic and puts the board into boot mode & they are not using QMK so it's not Bootmagic related. Also after flashing QMK , key is the same. So it is bootloadHID related (& I don't know why it's called a salt key* :)) 

It's documented in a video by somebody else here `https://youtu.be/9ha03F2rDWQ?t=277` and i've also found a more comprehensive README at the fork at `https://gitlab.h-kev.in/linux/qmk-firmware/blob/master/keyboards/jj4x4/README.md`

I'll come back also with a tap dance layout example later.

[*] https://github.com/qmk/qmk_firmware/blob/master/docs/flashing_bootloadhid.md

File: keyboards/jj4x4/README.md [hunk 3]
```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
 
     make jj4x4:default:flash
 
-**Reset Key**: Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard.
+**Reset Key**:
+
+Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard USB cable.
```
Comment (fauxpark) [inline keyboards/jj4x4/README.md]:
```suggestion
Hold down the key located at *K12*, commonly programmed as *8* while plugging in the keyboard USB cable.
```

My JJ4x4 arrived, it is indeed this key, but it is labelled K12 in the macro define.

File: keyboards/jj4x4/README.md [hunk 4]
```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
...
+
+Key *K00* is in the above picture:
```
Comment (fauxpark) [inline keyboards/jj4x4/README.md]:
```suggestion
Key *K12* is in the above picture:
```

File: keyboards/jj4x4/README.md [hunk 5]
```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
...
+
+Row 2 from the top, Column 2 from the left, see the schematic below:
+
+```
+  ,-----------------------.
+  |     |     |     |     |
+  |-----`-----`-----`-----|
+  |     | K00 |     |     |
```
Comment (fauxpark) [inline keyboards/jj4x4/README.md]:
```suggestion
  |     | K12 |     |     |
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**